content/en/continuous_integration/setup_pipelines/jenkins.md

```diff
@@ -222,20 +220,13 @@ If the CI Visibility option does not appear in the Datadog Plugin section, make
 
 ### The Plugin Tracer failed to initialized due to APM Java Tracer is used to instrument Jenkins
```

Comment:
Check the grammar here: "The plugin's tracer fails to initialize due to the APM Java tracer being used to instrument Jenkins"